manifests/init.pp

```diff
@@ -94,7 +97,11 @@
   Optional[String[1]]            $systemd_capability_bounding_set = undef,
   Optional[String[1]]            $systemd_ambient_capabilities    = undef,
   Optional[Boolean]              $systemd_no_new_privileges       = undef,
+  Stdlib::HTTPUrl                $caddy_url                       = 'https://caddyserver.com/download/linux',
```

Comment:
This was a static string, so went ahead and made it a param.

Reply:
This will be obsolete, because one of the open pull requests implements a new parameter called ```install_method```.  

Caddy 2 will be released next week and then the archive has to be downloaded from GitHub.
* https://caddyserver.com/v1/download
* https://github.com/caddyserver/caddy/releases

Reply:
Reverted this part.